diff --git a/node_export.module b/node_export.module
index 8bdadb9..d5f7764 100755
--- a/node_export.module
+++ b/node_export.module
@@ -1049,7 +1049,8 @@ function node_export_file_field_export(&$node, $original_node) {
               }
               // Remote export mode
               elseif ($export_mode == 'remote') {
-                $export_data = url($file->uri, array('absolute' => TRUE));
+                // NOTE: This is patched with info from https://drupal.org/node/2046431
+                $export_data = file_create_url($file->uri);
               }
               // Default is 'inline' export mode
               else {
